README.md

```diff
@@ -39,9 +41,9 @@ If you use an AMD bundling tool (like RequireJS):
 
 ```js
 define(["little-loader"], function (load) {
-  load("http://example.com/foo.js", function () {
-    // foo.js is loaded!
-  }/*, [optional context (`this`) variable here] */);
+  load("http://example.com/foo.js", function (err) {
+    // ... your code ...
+  });
```

Comment:
@baer -- It's still there and in the first example, just trying to cut down on repeated big comment pastes here and below.